src/server/snapshot.cc

```diff
@@ -20,6 +21,8 @@
 #include "server/tiered_storage.h"
 #include "util/fibers/synchronization.h"
 
+ABSL_FLAG(size_t, flush_big_entries_threshold, 0, "Total bytes before flushing big entries");
```

Comment:
Cause if we got any issues that we did not forsee it will make DF unstable by default. Furthermore, what we do here is a trade off between memory, latency and throughput. By serializing smaller chunks we reduce the memory footprint and provide a little better latency per packet (since we send them at smaller intervals). However, we reduce throughput because now we yield and we go back and forth. I would not like to have this defaulted. We could if we think it's best to have this on, add it on the release after this release

Reply:
> Cause if we got any issues that we did not forsee it will make DF unstable by default

If it's disabled, we won't know if it's unstable. We should build it the best that we can, and if there are bugs, we'll fix them :)

> Furthermore, what we do here is a trade off between memory, latency and throughput

One of the advantages of this feature is that actually Dragonfly will become more responsive to requests coming in while it is serializing. Today, when we serialize a huge entry, the entire thread is blocked, so for the duration of copying it (think 100mb, 1gb, even 10gb) we won't advance in the tx queue, won't even reply to pings coming from connections on this thread.
Latency of serialization is way less critical compared to this.